lib/baz/collection_events/collection_event.ex

```diff
@@ -45,15 +48,18 @@ defmodule Baz.CollectionEvents.CollectionEvent do
     |> cast(attrs, [
       :event_timestamp,
       :event_type,
-      :venue,
+      :id,
```

Comment:
Can we call this `venue_event_id`? I feel like I would get confused thinking it's the same as standard sequential ids.

Reply:
Sure.  Alternatively `source_id` or `native_id` ( shorter)?

Reply:
source_id it is!